resources/js/Shared/Widgets/UpcomingHolidays.vue

```diff
@@ -18,6 +18,9 @@
             </p>
             <p class="text-sm text-gray-500">
               {{ holiday.displayDate }} ({{ holiday.dayOfWeek }})
+              <span v-if="holiday.daysToHoliday !== 0">
+                - <strong>{{ holiday.daysToHoliday }}</strong>
```

Comment:
```suggestion
                - {{ holiday.daysToHoliday }}
```

I would drop this bolding.